targets/default.nix

```diff
@@ -37,6 +38,52 @@
     import ./nvidia-jetson-orin.nix {inherit jetpack-nixos microvm;}
     // {
       format = "raw-efi";
+      # XXX: This override is needed to reduce overall image size, but will be applied only for package and not for nixos configuration.
+      pkgs = let
+        overlay = self: super: {
+          qemu = super.qemu.override {
+            alsaSupport = false;
+            pulseSupport = false;
+            sdlSupport = false;
+            jackSupport = false;
+            gtkSupport = false;
+            vncSupport = false;
+            smartcardSupport = false;
+            spiceSupport = false;
+            ncursesSupport = false;
+            libiscsiSupport = false;
+            tpmSupport = false; # XXX: Probably shouldn't be disabled
+            hostCpuOnly = true;
+          };
+          git = super.git.override {
+            nlsSupport = false; # XXX: Propbably shouldn't
+            withManual = false;
+            pythonSupport = false;
+            withpcre2 = false;
+          };
+          systemd = super.systemd.override {
+            # withCompression = false; # XXX
+            withCryptsetup = false;
+            withDocumentation = false;
+            withFido2 = false;
+            withHomed = false;
+            withHwdb = false; # XXX
+            withLibBPF = true; # XXX: Probably should be disabled
+            withLocaled = false;
+            # withOomd = false; # XXX
+            withPCRE2 = false;
+            withPortabled = false; # XXX
+            # withResolved = false; # XXX
+            withShellCompletions = false;
+            withTpm2Tss = false;
+            withUserDb = false;
+          };
+        };
+      in
+        import nixpkgs {
+          localSystem = "aarch64-linux";
```

Comment:
Please also add `# TODO: enable also for x86-linux targets` here. We could do that in the next iterations but I'd like to test most minimal host on x86 targets as well - those are plenty more available and have generally better virtualization support on different VMMs than `aarch64`. From that perspective those are good benchmark for the `aarch64` reference work.

Reply:
With the latest change, this is no more a problem.